qa/tasks/vstart_runner.py

```diff
@@ -606,7 +606,7 @@ def raw_cluster_cmd_result(self, *args, **kwargs):
         proc = self.controller.run([os.path.join(BIN_PREFIX, "ceph")] + list(args), **kwargs)
         return proc.exitstatus
 
-    def admin_socket(self, daemon_type, daemon_id, command, check_status=True):
+    def admin_socket(self, daemon_type, daemon_id, command, check_status=True, timeout=None):
```

Comment:
Is the line related to this issue: http://tracker.ceph.com/issues/36581 ?

Reply:
It fixes those issue, but it's not a good idea to integrate this QA fix into this PR. Sadly this PR is already merged. Next time those issues had to split out into a separate issue because this thingy is not related to the origin PR.